src/Sulu/Bundle/MediaBundle/Tests/Fixtures/DefaultStructureCache.php

```diff
@@ -1,154 +0,0 @@
-<?php
-
-/*
- * This file is part of Sulu.
- *
- * (c) Sulu GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Sulu\Bundle\MediaBundle\Tests\Fixtures;
-
-use Sulu\Component\Content\Block\BlockProperty;
-use Sulu\Component\Content\Block\BlockPropertyType;
-use Sulu\Component\Content\Property;
-use Sulu\Component\Content\PropertyTag;
-use Sulu\Component\Content\Section\SectionProperty;
-
-/**
- * DefaultStructureCache.
- */
-class DefaultStructureCache extends \Sulu\Component\Content\Metadata
```

Comment:
the `\Sulu\Component\Content\Metadata` does not exist and i havent found out if there is a replacement for it. but it looks like this is not anyway 🙈

Reply:
as it is part of the `Tests` this can be safely be removed if its not used in any test longer